server/src/assets/game/tableGenerator/Readme.md

```diff
@@ -0,0 +1,35 @@
+# Table Generator for SWTbahn Game
+
+This python script generates the configuration json files for the swtbahn game client, which are required for the signal-symbol mapping and the route suggestion mechanics.
+The script will automatically find a suitable route (which the least nodes) and store it for the game client, it will do this action only for signals which are listed in the mapping-csv to reduce storage usage.
```

Comment:
"The script will automatically find a suitable route" -> to where? The latter part of the sentence implies it is done with all signals from the mapping-csv as the source, but it is not quite clear where these routes are leading.

"which the least nodes" -> grammar/incorrect english; rather say something like "find the shortest possible route from ... /(for all..) to ..." or something like that.